testing/containers/ceph/Dockerfile

```diff
@@ -13,11 +22,14 @@ RUN true && \
     yum clean all && \
     true
 
-ENV GOTAR=go1.15.10.linux-amd64.tar.gz
+ARG GO_VERSION=1.16.7
+ENV GO_VERSION=${GO_VERSION}
 RUN true && \
-    curl -o /tmp/${GOTAR} https://dl.google.com/go/${GOTAR} && \
-    tar -x -C /opt/ -f /tmp/${GOTAR} && \
-    rm -f /tmp/${GOTAR} && \
+    gotar=go1.15.10.linux-amd64.tar.gz && \
```

Comment:
There is still the go version hardcoded.

Reply:
Fixed.